tests/test_tensor.py

```diff
@@ -561,6 +561,8 @@ def test_chunk_same_size(self):
 
 
 class inequalityTest(unittest.TestCase):
+    # this is an ovverriden function from unittest.Testcase.
```

Comment:
Suggest a better way of writing this warning.

should we be doing something like 
```WARNING : message```